modules/internal/ChapelBase.chpl

```diff
@@ -1803,8 +1803,11 @@ module ChapelBase {
 
   // non-param/param and param/non-param
   // non-param/param version not necessary since > above works fine for that
+  inline proc >(param a: uint(64), b: uint(64)) param where a == 0 {
```

Comment:
Seems a little weird to have a param proc be marked inline too, but probably doesn't matter.

I wonder if this should be `param a: uint(?w)` and similar for b instead of uint(64). Doing that seems to allow the [test](https://github.com/chapel-lang/chapel/pull/5277/files#diff-c60cb09d88210e77db2f5c34662d065d) in https://github.com/chapel-lang/chapel/pull/5277 to pass

If so this might close https://github.com/chapel-lang/chapel/issues/5235, but there might be more @bradcray was looking to do there?

Reply:
I think if we can get that test to pass, we can close #5235.  I was trying to do something more aggressive in #5277, but haven't looked back at that in some time now...

Reply:
I have a lot yet to do this week so I'm going to consider these things future work (and not necessarily work for me). I made some notes in #5235.